.github/workflows/release.yml

```diff
@@ -31,6 +31,22 @@ jobs:
           GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
           GPG_PASSWORD: ${{ secrets.GPG_PASSWORD }}
 
+      - shell: bash
+        name: Collect SBOM's
+        run: |
+          mkdir sboms
+          find . -path ./sboms -prune -o -name "*.cyclonedx.json" -exec mv {} ./sboms/ \;
+          find . -path ./sboms -prune -o -name "*.spdx.json" -exec mv {} ./sboms/ \;
+          zip SBOM.zip sboms/*
+          cp SBOM.zip /tmp/
```

Comment:
Now that the boms are part of the maven publication, do we actually need to publish them as ``*.zip`?

Reply:
I think it makes sense to have a way to gather SBOM's consistently within OTel.  This way folks don't have to check NPM, Maven, etc to evaluate SBOM's across OTel.

Reply:
👍 seems reasonable to me. @open-telemetry/sig-security-maintainers can we codify this advice somewhere?